### PR TITLE
Add green tick to selected network - closes #935

### LINKED ISSUE #935: Add green tick to selected network
Screenshot 2022-02-09 at 15 01 18

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx [hunk 1]
```diff
@@ -28,34 +29,39 @@ const ChainSelectorPopup: FC<ChainSelectorPopupProps> = ({ open, onClose }) => {
...
       <div className="tc__chainSelectorPopup-content">
-        <div
-          className={`tc__chainSelectorPopup-button ${selectedChainName === 'ethereum' ? 'selected' : ''}`}
+        <button
+          className={`tc__chainSelectorPopup-button ${ethereumSelected ? 'selected' : ''}`}
           onClick={() => onChainSelect('ethereum')}
         >
-          <Typography variant="body-text">Ethereum Mainnet</Typography>
+          <div className="tc__chainSelectorPopup-button__label">
+            <Typography variant="button-text">Ethereum Mainnet</Typography>
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx:47]: I think we should remove `Mainnet` here?

File: packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx [hunk 2]
```diff
@@ -28,34 +29,39 @@ const ChainSelectorPopup: FC<ChainSelectorPopupProps> = ({ open, onClose }) => {
...
         <Spacer size={12} />
-        <div
-          className={`tc__chainSelectorPopup-button ${selectedChainName === 'fantom' ? 'selected' : ''}`}
+        <button
+          className={`tc__chainSelectorPopup-button ${fantomSelected ? 'selected' : ''}`}
           onClick={() => onChainSelect('fantom')}
         >
-          <Typography variant="body-text">Fantom Opera</Typography>
+          <div className="tc__chainSelectorPopup-button__label">
+            <Typography variant="button-text">Fantom Opera</Typography>
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx:59]: I think we should remove `Opera` here?

**Review by lazarmitic — CHANGES_REQUESTED** [review]
File SelectedIcon.tsx seems empty, I think we should remove it.

File: packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx [hunk 3]
```diff
@@ -28,34 +29,39 @@ const ChainSelectorPopup: FC<ChainSelectorPopupProps> = ({ open, onClose }) => {
     [selectedChain, onClose],
   );
 
+  const ethereumSelected = useMemo(() => selectedChainName === 'ethereum', [selectedChainName]);
```
[the remark this answers is not in the document]
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx:32]: ok

File: packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx:47]: Yes my bad

File: packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/navbar/ChainSelectorPopup.tsx:59]: done

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**